ulab_Crunch_Numbers_Fast/benchmark.py

```diff
@@ -1,11 +1,21 @@
-import time
+import utime
 import math
 import ulab
 import ulab.numerical
 
+def timeit(f, *args, **kwargs):
+    func_name = str(f).split(' ')[1]
+    def new_func(*args, **kwargs):
+        t = utime.ticks_us()
+        result = f(*args, **kwargs)
+        print('execution time: ', utime.ticks_diff(utime.ticks_us(), t), ' us')
+        return result
+    return new_func
+
 def mean(values):
     return sum(values) / len(values)
 
+@timeit
```

Comment:
I think @-decorators are probably unfamiliar to many CircuitPython users. Except for `@property`, `@attr.setter` and `@staticmethod`, it appears they are only used in one of the Python guides in Adafruit_Learning_System_Guides.  I don't require you to make this change, but consider whether it's important to structure it this way and if so maybe add a comment to the effect that "putting `@timeit` at the start of a function means that when it's called, the time it takes to run will be printed in the REPL" or words to that effect.

Reply:
I can definitely change this, I simply wasn't aware of the penetration of decorators. However, I find them quite useful, so if you think that `circuitpython` users are not familiar with the topic, this could be a good tutorial subject.